tools/depends/target/webos-userland/Makefile

```diff
@@ -1,5 +1,6 @@
 include ../../Makefile.include WEBOS-USERLAND-VERSION ../../download-files.include
 DEPS= ../../Makefile.include Makefile WEBOS-USERLAND-VERSION ../../download-files.include
+BASE_URL=https://github.com/webosbrew/webos-userland/archive/refs/tags/0.0.9
```

Comment:
uploaded to mirrors

Reply:
Thanks, dropped the temporary URL change commit